src/models/decisions/classification.ts

```diff
@@ -0,0 +1,26 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+import { Tag } from "../subject";
+import { IDecision } from "./decision";
+import { VerificationDecision } from "./verification";
+
+export interface IClassification {
+    tag: Tag;
+    confirmed: EnumValue<VerificationDecision>;
+    origin: number;
+}
+
+export class Classification implements IClassification, IDecision {
+    public constructor(data: IClassification) {
+        this.tag = data.tag;
+        this.confirmed = data.confirmed;
+        this.origin = data.origin;
+
+        this.identifier = Symbol("classification");
```

Comment:
this means no classification is ever equal to any other classification?

Reply:
Kind of.

I use this symbol to check if a decision has been applied to a tile or not